.babelrc

```diff
@@ -11,13 +11,8 @@
     ["@babel/typescript", { "allowNamespaces": true }]
   ],
   "plugins": [
-    "@babel/plugin-transform-modules-commonjs",
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
-    "@babel/plugin-proposal-class-properties",
-    "@babel/proposal-object-rest-spread",
-    "babel-plugin-styled-components",
-    "@babel/plugin-syntax-dynamic-import",
-    "babel-plugin-lodash"
+    "babel-plugin-styled-components"
```

Comment:
I'm not sure where this comment fits, but commit dfaf253adc added a `lib/Core/prerequisites.js` which seems to be about compiling to ES5 (which is also what Babel does?).

Should some things be removed from that file as well?